node/Cargo.toml

```diff
@@ -81,5 +85,9 @@ xyk-runtime-api = { default-features = false, version = '2.0.0', path = '../pall
 hex-literal = { version = "0.3.1", default-features = false }
 
 [features]
-default = []
+default = ['std']
```

Comment:
why do we need `std` here?

Reply:
Since root project is a virtual workspace (doesn't specify a package) we can't run it with features.
To run runtime-benchmark feature we need to run it from /node.
Node builds xyk-rpc. Mangata-runtime doesn't.
xyk-rpc needs to be built after xyk-runtime-api which is built by mangata-runtime.
Thus we need: std = [
		"mangata-runtime/std",
		"xyk-rpc/std",
]
We add default = ['std'] simply to avoid having to type
--features "runtime-benchmarks std"